src/App.js

```diff
@@ -33,10 +66,19 @@ class Image extends React.Component {
         window.removeEventListener("scroll", this.handleScroll);
     }
 
+    deleteCat = () => {
+        const countToDelete = this.state.countToDelete
+        this.setState((prevState) => {
+            const buffArray = prevState.images.slice(countToDelete, prevState.images.length);
```

Comment:
In this case you can omit the second argument for the `slice` function (https://www.w3schools.com/jsref/jsref_slice_array.asp).